packages/contentstack-auth/README.md

```diff
@@ -18,7 +18,7 @@ $ npm install -g @contentstack/cli-auth
 $ csdx COMMAND
 running command...
 $ csdx (--version)
-@contentstack/cli-auth/1.4.1 darwin-x64 node-v22.17.0
+@contentstack/cli-auth/1.4.1 darwin-arm64 node-v22.14.0
```

Comment:
If there is no change in the plugin, you don't need to push the README files too.